src/components/AvailabilityGrid.tsx

```diff
@@ -37,15 +50,19 @@ const Grid = ({
   }
 
   const [grid, setGrid] = useState(initialGrid)
-  const [isSelecting, setIsSelecting] = useState(false)
+  const [isSelecting, setIsSelecting] = useState(false) // When user is selecting cells
   const [dates, setDates] = useState<string[]>([])
+  const [hoveredCell, setHoveredCell] = useState<{
+    // hovered cell on grid used for styling
+    rowIndex: number
+    colIndex: number
+  } | null>(null)
 
-  // Update grid dimensions when daysOfWeek changes
+  // Populate the grid with creator's availability times saved in the database (view-event)
   useEffect(() => {
-    setGrid(initialGrid())
     if (mode === 'weekly') {
       const order = ['Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat']
-      config = config.sort((a, b) => order.indexOf(a) - order.indexOf(b))
+      //const sortedConfig = config.sort((a, b) => order.indexOf(a) - order.indexOf(b))
```

Comment:
commented out on purpose?

Reply:
having issues with this line where sorting the days causes misalignments with responses and time availabilities on the grid. Leaving this commented out until we have a fix for now